docs/source/events/payment-status-change.rst

```diff
@@ -6,6 +6,30 @@ This event means that the status of a payment has changed. To reduce noise, we o
 For each payment method, there are typical status sequences that the payment goes through. The exact meaning of the statuses and content of the ``payment_data`` also depends on the payment method.
 
 
+Line items
+----------
+
+.. versionadded:: 1.2.0
+
+Each status update contains the full list of payment ``line_items``. Each line item has the following data:
+
+- ``name``: A unique name for this line item
+- ``amount`` (float): The price of each unit
+- ``quantity`` (float): The number of items
+- ``tax_rate`` (float): The applicable VAT (or equivalent) tax rate
+- ``recurrence_interval`` (interval): For recurring payments this specifies the interval (e.g. ``P1M`` for monthly, ``P1Y`` for yearly). It is set to ``null`` for one-off payments.
```

Comment:
I suggested examples here earlier. This looks fine. Not sure how it'll look on the webpage from these docs but should be ok?

Reply:
I’ve posted a screenshot above.